package/arch/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN    pacman -Sy   --noconfirm archlinux-keyring \
                        openssh                    \
                        python                     \
                        python-pip                 \
+                       python-poetry              \
```

Comment:
The `makedepends` in `package/arch/PKGBUILD` should probably be updated with this build dependency.

Reply:
Feel free to add back automerge.

Reply:
Done.